.github/workflows/e2e_tests.yml

```
@@ -3,62 +3,115 @@ name: E2E Tests
+        working-directory: 'frontend/internal-packages/e2e'
     env:
       CI: true
       URL: ${{ github.event.deployment_status.target_url }}
       ENVIRONMENT: ${{ github.event.deployment.environment }}
+
     steps:
       - name: Checkout repository
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
         with:
-          ref: ${{ github.event.deployment.sha }}
+          ref: ${{ steps.snapshot.outputs.sha }}
```

actionlint (1.7.7)

property "snapshot" is not defined in object type {}

(expression)